test/unit-tests.js

```
@@ -415,9 +396,6 @@ Object.keys(cases).forEach(function (key) {
                 testError(esprima, testCase.case, JSON.parse(testCase.failure));
             } else if (testCase.hasOwnProperty('result')) {
                 testAPI(esprima, testCase.run, JSON.parse(testCase.result));
-            } else {
-                console.error('Incomplete test case:' + testCase.key + '. Generating test result...');
```

will this now silently fail?